app/controllers/concerns/current_user_concern.rb

```diff
@@ -0,0 +1,11 @@
+module CurrentUserConcern
+  extend ActiveSupport::Concern
+
+  included do
+    private
+
+    def user
+      @user ||= User::Current.new(session['user_id'])
```

Comment:
@developish - I'm tempted to suggest that we move this method into ApplicationController, instead of including it in each controller

Reply:
I just realized this is a QA PR, and that Marc reviewed the other one already.